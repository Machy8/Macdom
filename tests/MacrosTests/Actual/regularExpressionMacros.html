https://code.jquery.com/jquery-3.2.1.js
https://code.jquery.com/jquery-3.2.1.js async
path/to/css/file.css

some text with path/to/css/file.css