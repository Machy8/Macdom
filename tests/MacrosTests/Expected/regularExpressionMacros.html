<script src="https://code.jquery.com/jquery-3.2.1.js" type="text/javascript"></script>
<script src="https://code.jquery.com/jquery-3.2.1.js" type="text/javascript" async></script>
<link href="path/to/css/file.css" rel="stylesheet" type="text/css">
some text with path/to/css/file.css
